### PR TITLE
Add EVA species rabbit oryctolagus_cuniculus

### PR DESCRIPTION
**Pull request by ima23**
Adding EVA species that use a VCF as source of variation: ENSVAR-3908
Sandbox using config: http://ves-hx2-76.ebi.ac.uk:6080/Oryctolagus_cuniculus/Info/Index
Example variant link on the above page: 2:6565980:G_C:PRJEB27278

### CODE REVIEW

File: ensembl/conf/ini-files/oryctolagus_cuniculus.ini [hunk 1]
```diff
@@ -65,8 +65,10 @@ ENSEMBL_SEARCH_IDXS = [ Domain Family Gene GenomicAlignment Marker Sequence ]
 
 SEARCH_TEXT       = glycoprotein
 
+VARIATION_PARAM   = 2:6565980:G_C:PRJEB27278;vf=2:6565980:G_C:PRJEB27278
+VARIATION_TEXT    = 2:6565980:G_C:PRJEB27278
+
 ENSEMBL_SOUND     = Eh, what's up, doc?
 VEP_HGVS          = ENSOCUT00000014253.2:c.4C>A\nENSOCUT00000008071.3:c.168+16T>C\nENSOCUT00000016986.3:c.5delA
```
Comment (at7) [inline ensembl/conf/ini-files/oryctolagus_cuniculus.ini:72]: Could you please regenerate the VEP examples. The transcript versions are outdated. It is best to also make sure that the VEP examples are up-to-date for new VCF species. I already updated the documentation. Thank you!

**Review by at7 — APPROVED** [review]
Thank you. Looks all good now.

File: ensembl/conf/ini-files/oryctolagus_cuniculus_vcf.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      "species": "oryctolagus_cuniculus",
+      "assembly": "OryCun2.0",
+      "type": "remote",
+      "use_as_source" : 1,
+      "use_seq_region_synonyms": 1,
+      "filename_template" : "ftp://ftp.ebi.ac.uk/pub/databases/eva/PRJEB27278/Craniosnyostotic_Rabbit_Colony_eva08062018.vcf.gz",
+      "chromosomes": [
+        "1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","X"
+      ]
+    }
```
Comment (at7) [inline ensembl/conf/ini-files/oryctolagus_cuniculus_vcf.json:16]: could you please add:
{
      "id": "90_mammals.gerp_conservation_score",
      "species": "oryctolagus_cuniculus",
      "assembly": "OryCun2.0",
      "type": "remote",
      "filename_template" : "/90_mammals.gerp_conservation_score/gerp_conservation_scores.oryctolagus_cuniculus.OryCun2.0.bw",
      "annotation_type": "gerp"
    }

**Review by at7 — APPROVED** [review]

**Comment by ens-st3** [comment]
Can you remind me what is the general approach for adding examples to DBs vs .INI files ? Example variants go into DBs (apart from Human), VEP examples in .INI files ?

**Comment by ima23** [comment]
@ens-st3 : we add variant examples and VEP examples to .INI files. There are future plans for production to automatically add examples to the DBs and we to curate/update them for human, farm animals, species of high interest. Not sure what is the ETA for that automation.

**Pull request merged**